### PR TITLE
Rename `platformContributionAmount` to `platformTipAmount`

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5052

⚠️ Requires: https://github.com/opencollective/opencollective-api/pull/6979

### CODE REVIEW

File: components/contribution-flow/index.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ class ContributionFlow extends React.Component {
                 : null,
             customData: stepDetails.customData,
             paymentMethod: await this.getPaymentMethod(),
-            platformContributionAmount: getGQLV2AmountInput(stepDetails.platformContribution, undefined),
+            platformTipAmount: getGQLV2AmountInput(stepDetails.platformContribution, undefined),
```
Comment (znarf) [inline components/contribution-flow/index.js:204]: As it's really sensible, can you point me again on where it's supported in the API?

Comment (SudharakaP) [reply components/contribution-flow/index.js:204]: This is where we added the `platformTipAmount` when creating the order; 

https://github.com/opencollective/opencollective-api/blob/58e91edad9f836b7fb1fc913ea79517a2fa2e450/server/graphql/v2/mutation/OrderMutations.js#L72

Also just to note this is merged we have one final PR; https://github.com/opencollective/opencollective-api/pull/6980 to rebase and merge to remove the older platformContributionAmount

**Review by znarf — APPROVED** [review]

**Comment by znarf** [comment]
When this lands production, please ensure that it works as expected and let me know when that's done.

**Comment by SudharakaP** [comment]
> When this lands production, please ensure that it works as expected and let me know when that's done.

Deployed to prod and tested with a platform tip contribution to https://opencollective.com/opencollective and seems to work fine. 👍🏽

**Pull request merged**